src/platform/plugins/shared/unified_doc_viewer/public/components/observability/traces/components/similar_spans/index.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { DurationDistributionChart } from '@kbn/apm-ui-shared';
+import { ProcessorEvent } from '@kbn/apm-types-shared';
+import { ContentFrameworkChart } from '../../../../content_framework/chart';
+import { ContentFrameworkSection } from '../../../../content_framework/section';
+import type { SpanLatencyChartData } from '../../doc_viewer_span_overview/hooks/use_span_latency_chart';
+
+export interface SimilarSpansProps {
+  spanDuration: number;
+  latencyChart: {
+    data: SpanLatencyChartData | null; // TODO move this interface
+    loading: boolean;
+    hasError: boolean;
+  };
+  isOtelSpan: boolean;
+  esqlQuery?: string;
+}
+
+// This section will be replacing the SpanDurationSummary and TransactionDurationSummary as part of https://github.com/elastic/kibana/issues/228916
+export function SimilarSpans({
+  latencyChart,
+  spanDuration,
+  esqlQuery,
+  isOtelSpan,
+}: SimilarSpansProps) {
+  return (
+    <ContentFrameworkSection
+      id="similarSpans"
+      data-test-subj={'docViewerSimilarSpansSection'}
+      title={i18n.translate('unifiedDocViewer.observability.traces.similarSpans', {
+        defaultMessage: 'Similar spans',
+      })}
+    >
+      <ContentFrameworkChart
+        data-test-subj={`docViewerSimilarSpansLatencyChart`}
+        title={i18n.translate('unifiedDocViewer.observability.traces.similarSpans.latency.title', {
+          defaultMessage: 'Latency',
+        })}
+        esqlQuery={!latencyChart.hasError && esqlQuery ? esqlQuery : undefined}
+      >
+        {(latencyChart.loading || latencyChart.data) && (
```

Comment:
Do we need this condition? Looks like the `DurationDistributionChart` could handle internally if either is undefined